src/Data/Form.purs

```diff
@@ -138,23 +58,18 @@ type InputState t r =
   )
 
 data Input f
-  = Text (Record (SharedRows f String + RequiredRows f + StringRows f ()))
-  | Toggle (Record (SharedRows f Boolean ()))
+  = Text (Record (SharedRows f + RequiredRows f + StringRows f ()))
+  | Toggle (Record (SharedRows f ()))
 
-derive instance genericInput :: Generic (Input f) _
+derive instance genericInput :: Generic (Input Expr) _
```

Comment:
I changed this on accident. Maybe we can remove the change so the diff is a little cleaner?

```suggestion
derive instance genericInput :: Generic (Input f) _
```